docs/book/installation.rst

```diff
@@ -61,7 +61,7 @@ And now you can use gulp for installing views, by just running a simple command:
 
 .. code-block:: bash
 
-    $ npm run gulp
+    $ gulp
```

Comment:
It is not only about where gulp is installed, but also that the previous command was the only way, to run a npm script. otherwise we would have to call a binary from a node_modules or link this to bin folder. But of course you are 💯 % right.